src/Pages/signin/Components/SignInForm.tsx

```diff
@@ -1,22 +1,80 @@
 import React from "react";
-import Box from "@material-ui/core/Box";
-import Typography from "@material-ui/core/Typography";
+import styled from "styled-components";
+import Background from "../../../assets/StrokeBackground.png";
+import LoginLogo from "../../../assets/LoginLogo.png";
+//React router
+import {Link} from 'react-router-dom';
+//color
+export const colors = {
+  grey: "#D3D3D3",
+  pink:"#EF5DA8",
+  white:"#FFFFFF"
+};
+
+//background
+export const StyledContainer = styled.div`
+  margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0)),
+    url(${Background});
+  background-size: cover;
+  background-attachment: fixed;
+`;
+
+//loginbox
+export const StyledFormArea = styled.div`
+  position: absolute;
+  width: 500px;
+  height: 400px;
+  left: 500px;
```

Comment:
For line 32 and 33, if you set `display: flex` , it will be center of the page automatically (maybe). Because, the parent component(StyledContainer) has set display rules to flex , justify-content, and align-items to center already.